guidelines.py

```diff
@@ -57,24 +57,26 @@
     0x7ffe00, 0xfe007f, 0xfe7ffe
 ])
 
+server_url = 'https://discord.com/channels/323076998576603137/'
```

Comment:
Code duplicate, part of this string is defined already at `channel_url`. The following diff would had been better.

```diff
-channel_url = 'https://discord.com/channels/323076998576603137/860920579154772018/'
+server_url = 'https://discord.com/channels/323076998576603137/'
+channel_url = f'{server_url}860920579154772018/'

Reply:
already fixed in 1136b29